exercises/practice/high-scores/high_scores.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+namespace arcade {
+
+class HighScores {
+   private:
+    std::vector<int> scores;
+
+   public:
+    HighScores(std::vector<int> scores) : scores(scores){};
```

Comment:
just to avoid that this question is getting lost among the others:
Do we break with the tradition and provide the implementation of the constructor?  
In all other exercises, even the ones with where we provide some declarations or some empty functions in the initial code we still leave the implementation of all functions to the students.

Reply:
My intention is "the header file does not need to be changed". I would prefer to hand over an idiomatic file that has the constructor defined, to skip it on purpose and give the impression, that it has to go into the source file.

I did not know that there was some tradition to the degree we hand over the scaffolding. May exercises are super barebones and as a student, it felt more like a chore to fill in the header file from the test file content. Especially since nothing compiles if you skip it.

If we decide to leave out the constructor definition, there will be an unused variable warning and the code will not compile, unless I fix the CMakeLists.txt accordingly. I had the plan to keep those changes to the concepts.